src/platyPS/platyPS.psm1

```diff
@@ -656,6 +656,7 @@ function New-MarkdownAboutHelp
 
     begin
     {
+        if ($AboutName.StartsWith('about_')) {$AboutName = $AboutName.Substring(6,$($AboutName.Length - 6))}
```

Comment:
no need to pass the second argument: there is an override that take 1 argument and takes the substring until the end.
Can we avoid hard-coded magic number 6? I.e. `'about_'.Length`